docs/advanced-features/athena.ipynb

```diff
@@ -0,0 +1,688 @@
+    "\n",
+    "1. From \"Admin Settings\", scroll to \"Roles\"\n",
+    "2. Click on the \"+\" to create a new Role\n",
+    "3. Set Name to e.g., \"AthenaAccessRole\"\n",
+    "4. Click on \"No policies attached.  Attach a policy…\"\n",
+    "5. Select the \"AthenaQuiltAccess\" policy from before\n",
+    "6. Click \"Create\"\n",
+    "\n",
+    "See [Users and roles](../Catalog/Admin.md) for more details on access control management in Quilt."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6909243d-27b8-4be2-93b4-bda0d26514e5",
+   "metadata": {},
+   "source": [
+    "## III. Defining Per-Bucket Metadata Tables in Athena\n",
+    "The next step is enabling Athena to query the package contents and metadata\n",
+    "for a specific Quilt bucket, by creating proxy tables and views that represent those files:\n",
+    "<!--pytest-codeblocks:cont-->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "f3f532d0-62e3-4aa2-a682-0f4cbeaac663",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "QUILT_BUCKET=\"quilt-example\"   # Use your own\n",
+    "QUILT_URL=\"s3://\"+QUILT_BUCKET # With S3 Prefix\n",
+    "\n",
+    "BUCKET_ID=QUILT_BUCKET.replace('-','_')\n",
+    "MANIFEST_TABLE=f\"{BUCKET_ID}_quilt_manifests\"\n",
+    "PACKAGES_TABLE=f\"{BUCKET_ID}_quilt_packages\"\n",
+    "PACKAGES_VIEW=f\"{BUCKET_ID}_quilt_packages_view\"\n",
+    "OBJECTS_VIEW=f\"{BUCKET_ID}_quilt_objects_view\"\n",
+    "DDL = {}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2179997c-399b-43a8-b280-aed4cf411e8a",
+   "metadata": {},
+   "source": [
+    "### A. Manifests table\n",
+    "The following Athena DDL will build a table of all the manifests in that bucket\n",
+    "(all package-level and object-level metadata).\n",
+    "<!--pytest-codeblocks:cont-->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "e9b4d2fa-6463-465b-8072-db45466ed596",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DDL[MANIFEST_TABLE] = f\"\"\"\n",
+    "CREATE EXTERNAL TABLE IF NOT EXISTS `{ATHENA_DB}.{MANIFEST_TABLE}`(\n",
+    "  `logical_key` string,\n",
+    "  `physical_keys` array<string>,\n",
+    "  `size` string,\n",
+    "  `hash` struct<type:string,value:string>,\n",
+    "  `meta` string,\n",
+    "  `user_meta` string,\n",
+    "  `message` string,\n",
+    "  `version` string)\n",
+    "ROW FORMAT SERDE\n",
+    "  'org.openx.data.jsonserde.JsonSerDe'\n",
+    "WITH SERDEPROPERTIES (\n",
+    "  'ignore.malformed.json'='true')\n",
+    "STORED AS INPUTFORMAT\n",
+    "  'org.apache.hadoop.mapred.TextInputFormat'\n",
+    "OUTPUTFORMAT\n",
+    "  'org.apache.hadoop.hive.ql.io.IgnoreKeyTextOutputFormat'\n",
+    "LOCATION\n",
+    "  '{QUILT_URL}/.quilt/packages'\n",
+    "TBLPROPERTIES (\n",
+    "  'has_encrypted_data'='false',\n",
+    "  'transient_lastDdlTime'='1605312102')\n",
```

Comment:
I think this can be omitted and `transient_lastDdlTime` probably even should be omitted.

Reply:
While perhaps not necessary for the query, I think it is useful to have in an example, so that users know the properties exist.